src/Castle.Core.Tests/ModuleScopeTestCase.cs

```diff
@@ -202,7 +208,7 @@ private static void CheckUnsignedSavedAssembly(string path)
 		{
 			Assert.IsTrue(File.Exists(path));
 
-			var assemblyName = AssemblyName.GetAssemblyName(path);
+			var assemblyName = new AssemblyName(Path.GetFileNameWithoutExtension(path));

```

Comment:
This change doesn't seem quite right since the file name isn't the assembly name. I assume the test passes because it works in the scenario for this test?